tests/unit/conftest.py

```diff
@@ -55,3 +61,54 @@ def session_error(self: SQLAlchemy) -> None:
             yield
         finally:
             SQLAlchemy.session = original_session_property  # type: ignore[method-assign]
+
+
+@pytest.fixture(scope="function")
+def submission_awaiting_sign_off(factories: _Factories) -> Generator[Submission, None, None]:
+    grant_recipient = factories.grant_recipient.build(
+        grant__name="Test grant",
+    )
+
+    question = factories.question.build(
+        form__collection__grant=grant_recipient.grant,
+        form__collection__name="Test collection",
+        form__collection__submission_period_end_date=date(
+            2025,
+            12,
+            3,
+        ),
+    )
+    submission = factories.submission.build(
+        grant_recipient=grant_recipient,
+        collection=question.form.collection,
+        mode=SubmissionModeEnum.LIVE,
+        data={str(question.id): "Question answer"},
+    )
+    submitted_by = factories.user.build(name="Submitter User", email="submitter@test.com")
+
+    form = submission.collection.forms[0]
+
+    factories.submission_event.build(
+        submission=submission,
+        related_entity_id=form.id,
+        event_type=SubmissionEventType.FORM_RUNNER_FORM_COMPLETED,
+        created_by=submitted_by,
+        created_at_utc=datetime(2025, 11, 25, 11, 0, 0),
+    )
+    factories.submission_event.build(
+        submission=submission,
+        event_type=SubmissionEventType.SUBMISSION_SENT_FOR_CERTIFICATION,
+        created_by=submitted_by,
+        created_at_utc=datetime(2025, 11, 26, 13, 30, 0),
+    )
+
+    yield submission
+
+
+@pytest.fixture(scope="function")
+def mock_send_email(mocker: MockerFixture) -> Generator[Any, Any, None]:
```

Comment:
This feels like a close duplicate of mock_notification_service_calls, is it distinct+useful enough to have this?